app/views/stream_entries/_simple_status.html.haml

```diff
@@ -1,7 +1,7 @@
 .status.light
   .status__header
     .status__meta
-      = link_to time_ago_in_words(status.created_at), TagManager.instance.url_for(status), class: 'status__relative-time u-url u-uid', title: l(status.created_at), target: stream_link_target, rel: 'noopener'
+      = link_to local_time_ago(status.created_at), TagManager.instance.url_for(status), class: 'status__relative-time u-url u-uid', title: l(status.created_at), target: stream_link_target, rel: 'noopener'
```

Comment:
It might be worth noting that the format used in the stream entries one here is a slight format change.

Currently everything is like "13m", "39m", "3h", "10h", "1d", "2d", ... and I think never does more than the days ago count of n in a string like "nd".

The change here would behave differently, but has options: https://github.com/basecamp/local_time/blob/master/README.md#time-ago-helper